documentation/src/docs/asciidoc/release-notes/release-notes-6.0.0-M1.adoc

```diff
@@ -115,6 +115,21 @@ repository on GitHub.
 * The contracts for the `Executable` parameters of Kotlin-specific `assertTimeout`
   functions were changed from `callsInPlace(executable, EXACTLY_ONCE)` to
   `callsInPlace(executable, AT_MOST_ONCE)` which might result in compilation errors.
+* The `CsvFileSource.lineSeparator()` parameter has been removed. The line separator
+  is now automatically detected, meaning that any of `\r`, `\n`, or `\r\n` is
+  treated as a line separator.
+* As a result of migrating from
+  https://github.com/uniVocity/univocity-parsers[univocity-parsers] to
+  https://fastcsv.org/[FastCSV] for CSV input handling, the root causes and messages
+  of exceptions thrown when parsing malformed CSV input may differ in rare cases.
+  While overall parsing behavior remains consistent, this may affect
+  custom error handling that relies on specific exception types or messages.
+* Parameters such as `ignoreLeadingAndTrailingWhitespace()`, `nullValues()`,
+  and others in `CsvSource` and `CsvFileSource` now apply to header fields as well
+  as to regular fields.
+* Empty lines in `CsvSource.value()` no longer cause a
+  `PreconditionViolationException`; such lines are now ignored, consistent with
+  the behavior of `CsvSource.textBlock()`.
```

Comment:
Did we discuss this? I think blank lines in a text block may be used to increase readability while passing blank strings to `@CsvSource` looks fishy. :thinking: